great_expectations/experimental/datasources/file_path_data_asset.py

```diff
@@ -53,6 +56,10 @@ class _FilePathDataAsset(DataAsset):
         "kwargs",  # kwargs need to be unpacked and passed separately
     }
 
+    _ALWAYS_INCLUDE_IN_BATCH_REQUEST_OPTIONS_TEMPLATE: ClassVar[Set[str]] = {
```

Comment:
@NathanFarmer I found a way to handle this using `batch_spec` (in #7197).  It would be great if we could merge this pull request after that one; otherwise, the merge can become very tedious (due to the refactoring).  Thank you.

Reply:
@alexsherstinsky I am not inclined to wait any longer to merge another one of your PRs.